### PR TITLE
Docs: promoting to production

### PR DESCRIPTION
**Pull request by anayeaye**
## What
This is phase one of upgrading our ingestion documentation. The goal is to update the veda-docs/contributing pages with current links and examples. More changes for automation are planned soon (#161)  but should hopefully only require a small update to the README. https://github.com/NASA-IMPACT/veda-data/pull/162/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R149

> **NOTE** I moved the geoglam ingestion example to veda-docs. The README link to this notebook should be confirmed after the corresponding veda-docs PR is merged (https://github.com/NASA-IMPACT/veda-docs/pull/161)

### CODE REVIEW

File: transformation-scripts/example-template/example-geoglam-ingest.ipynb [hunk 1]
```diff
@@ -76,10 +76,15 @@
    "source": [
```
Comment (anayeaye) [inline transformation-scripts/example-template/example-geoglam-ingest.ipynb:76]: Should this notebook be moved to veda-docs?

Comment (smohiudd) [reply transformation-scripts/example-template/example-geoglam-ingest.ipynb:76]: Agree that this should move to veda-docs

File: README.md [hunk 2]
```diff
@@ -170,11 +177,40 @@ The `ingestion-data/dataset-config/` directory holds json files that can be used
...
+    "filename_regex": "<filename-regex>",
+
+    ### misc
+    "dry_run": "<true/false>"
+}
 ```
 
+</details>
+
 ## Validation
```
Comment (smohiudd) [inline README.md:211]: @anayeaye as part of the validation should we also create json schema that includes our minimum requirements for collection metadata to maintain consistency across the production catalog? For example, all collections in VEDA must include:

- providers
- thumbnails assets

Comment (anayeaye) [reply README.md:211]: Ooof I just included those there in an optional section in the updated geoglam notebook that is now runnable in the hub https://github.com/NASA-IMPACT/veda-docs/blob/c9dc33a8f2960fa29a59db2974f979b12055f8ca/contributing/dataset-ingestion/example-template/example-geoglam-ingest.ipynb

Comment (anayeaye) [reply README.md:211]: I don't disagree on a firm nudge to create renders and thumbnails but I don't think we should require them until we have great docs on how to set them up + a way for users to upload thumbs (which are both doable)

**Review by smohiudd — APPROVED** [review]

**Pull request merged**